src/aw-client.ts

```diff
@@ -82,32 +93,35 @@ export class AWClient {
         } else {
             this.baseURL = options.baseURL;
         }
+        this.apiURL = this.baseURL + "/api";
         this.controller = options.controller || new AbortController();
-
-        this.req = axios.create({
-            baseURL: this.baseURL + "/api",
-            timeout: options.timeout || 30000,
-        });
     }
 
     private async _get(endpoint: string, params: object = {}) {
-        return this.req
-            .get(endpoint, { ...params, signal: this.controller.signal })
-            .then((res) => (res && res.data) || res);
+        return fetchWithFailure(`${this.apiURL}${endpoint}`, {
+            ...params,
+            signal: this.controller.signal,
+        });
```

Comment:
Any `_get` should return JSON, so just return it here instead of doing `.then((res) => res.json())` everywhere.